src/psyclone/dynamo0p3.py

```diff
@@ -7677,6 +7758,15 @@ def base_name(self):
         '''
         return self._base_name
 
+    @property
+    def argument_kinds(self):
+        '''
+        :returns: kinds (precisions) for all arguments in a kernel.
+        :rtype: set
```

Comment:
Please also specify the type of the members of the set.

Reply:
Done.